shared/src/test/diff/parser/NegativeLits.mls

```diff
@@ -1,14 +1,18 @@
 :NewParser
 :ParseOnly
 
+:e // FIXME: What does this test represent?
```

Comment:
Oh, this test makes me realize why @chengluyu added support for negative literals in the lexer in the first place – it was to support negative singleton types like `-1`, as demonstrated here.

A better solution is to add a case in the code you added to the parser: check when the rhs is an integer literal, and create a new negative integer literal as a result.